R-Interface/jasprcpp.cpp

```diff
@@ -65,32 +66,84 @@ static pollMessagesFuncDef		_pollMessagesFunction	= nullptr;
 static systemDef				_systemFunc				= nullptr;
 static libraryFixerDef			_libraryFixerFunc		= nullptr;
 static std::string				_R_HOME = "";
+static bool						_insideJASP				= true;
 
 bool shouldCrashSoon = false; //Simply here to allow a developer to force a crash
 
 //Ugly hack to work around windows messing up environment variables when local+codepage+utf8
 //Might not be necessary anymore due to the active codepage being set to utf8 now
 extern char * R_TempDir;
 
+// Code from RInside.cpp
```

Comment:
Couldve been in you know, another file...
Cause jasprcpp is getting more and more readable by the year :p

Also, you are still linking to Rinside?
So why copy this code?